src/util/ReactUtils.ts

```diff
@@ -135,7 +135,7 @@ export const findChildByType = (
 /*
  * Create a new array of children excluding the ones matched the type
  */
-export const withoutType = (children: ReactNode, type: string) => {
```

Comment:
This type fix reflects the usage of the function, as well as the actual implementation.